qelem/lib/screens/register_login_screen.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:flutter/material.dart';
+
+class RegisterLoginScreen extends StatelessWidget {
+  const RegisterLoginScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final pads = MediaQuery.of(context).size.height / 15;
+    final buttonHeight = MediaQuery.of(context).size.height / 15;
+    final imageHeight = MediaQuery.of(context).size.height / 2;
+    final mainWelcomeTextStyle = Theme.of(context).textTheme.titleMedium;
+    final additionalIntrosStyle = Theme.of(context).textTheme.subtitle1;
+    const signInButtonText = "SIGN IN";
+    const createAccButtonText = "CREATE AN ACCOUNT";
+    const imagePath = "";
+    const mainWelcomeText = "Hey! Welcome";
+    const additionalIntros =
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam egestas rhoncus lectus rhoncus, tempor. ";
+
+    return Scaffold(
+      body: ListView(
+        children: [
+          SizedBox(height: pads),
+          Container(
+            padding: EdgeInsets.all(pads),
+            width: double.infinity,
+            height: imageHeight,
+            child: const Image(
+              image: AssetImage(imagePath),
+              fit: BoxFit.fill,
+            ),
+          ),
+          Column(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Container(
+                padding: EdgeInsets.only(left: pads, right: pads),
+                width: double.infinity,
+                height: buttonHeight + 2 * pads,
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.spaceAround,
+                  children: [
+                    Text(
+                      mainWelcomeText,
+                      style: mainWelcomeTextStyle,
+                      textAlign: TextAlign.center,
+                    ),
+                    Text(
+                      additionalIntros,
+                      textAlign: TextAlign.center,
+                      style: additionalIntrosStyle,
+                    ),
+                  ],
+                ),
+              ),
+              Container(
+                padding: EdgeInsets.symmetric(horizontal: pads),
+                width: double.infinity,
+                height: buttonHeight + 2 * pads,
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.spaceAround,
+                  children: [
+                    ElevatedButton(
+                      style: ElevatedButton.styleFrom(
+                        primary: Theme.of(context).primaryColor,
+                        minimumSize: Size(double.infinity, buttonHeight),
+                      ),
```

Comment:
Just use the default style, there is nothing special about these buttons. We will customize them from the material app theme. The same goes for the other button.

Reply:
what do you mean the default style?

Reply:
The style from the material app theme, that is inherited by default.